code/game/atoms_movable.dm

```diff
@@ -25,6 +25,8 @@
 		crash_with("GC: -- [type] was deleted via del() rather than qdel() --") // stick a stack trace in the runtime logs
 	..()*/
 
+	var/z_levels_fallen = 0

```

Comment:
Pls move this up above the commented out Del() proc & perhaps change it to `var/tmp/`.